cscs-checks/system/slurm/slurm_check.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+# TODO: scontrol, do we want to scontrol something specific?
+@rfm.simple_test
+class SlurmCheck(rfm.RunOnlyRegressionTest):
+    slurm_command = parameter(['squeue', 'sacct', 'sinfo', 'scontrol'])
+
+    def __init__(self):
+        self.descr = 'Slurm command test'
+        self.valid_systems = ['daint:login', 'dom:login']
+        self.valid_prog_environs = ['builtin']
+        self.num_tasks = 1
+        self.num_tasks_per_node = 1
+        self.sanity_patterns = sn.assert_found(r'0', self.stdout)
```

Comment:
This will change when you remove the `echo $?`

Reply:
Done